src/dft/src/cells/OneBitScanCell.hh

```diff
@@ -44,19 +46,25 @@ class OneBitScanCell : public ScanCell
   OneBitScanCell(const std::string& name,
                  std::unique_ptr<ClockDomain> clock_domain,
                  odb::dbInst* inst,
-                 sta::TestCell* test_cell);
+                 sta::TestCell* test_cell,
+                 sta::dbNetwork* db_network,
+                 utl::Logger* logger);
   // Not copyable or movable
   OneBitScanCell(const OneBitScanCell&) = delete;
   OneBitScanCell& operator=(const OneBitScanCell&) = delete;
 
   uint64_t getBits() const override;
-  void connectScanEnable() const override;
-  void connectScanIn() const override;
-  void connectScanOut() const override;
+  void connectScanEnable(const ScanDriver& driver) const override;
+  void connectScanIn(const ScanDriver& driver) const override;
+  void connectScanOut(const ScanLoad& load) const override;
+  ScanDriver getScanOut() const override;
 
  private:
   odb::dbInst* inst_;
   sta::TestCell* test_cell_;
+  sta::dbNetwork* db_network_;
+
+  odb::dbITerm* findITerm(sta::LibertyPort* liberty_port) const;
```

Comment:
member functions before data members per https://google.github.io/styleguide/cppguide.html#Declaration_Order

Reply:
Thanks!, I fixed other classes that were not correct.